aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js

```diff
@@ -0,0 +1,321 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Grid,
+  Header,
+  Heading,
+  List,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import {
+  Cluster,
+  Group,
+  Hpe,
+  List as ListIcon,
+  Location,
+  LocationPin,
+  Optimize,
+  Overview,
+  Server,
+  Servers,
+  ShieldSecurity,
+  UnorderedList,
+} from 'grommet-icons';
+
+// Mock application page data
+const pages = [
+  {
+    name: 'Home',
+    layout: 'tiles',
+    children: ['Devices', 'Performance', 'Users', 'Remote Access', 'Security'],
+  },
+  {
+    name: 'Performance',
+    icon: <Optimize />,
+    layout: 'tiles',
+    parent: 'Home',
+    children: ['Sessions', 'Logs', 'Diagnostics'],
+  },
+  {
+    name: 'Devices',
+    icon: <Servers />,
+    layout: 'tiles',
+    parent: 'Home',
+    children: ['Server Cluster', 'Beacon Cluster'],
+  },
+  {
+    name: 'Users',
+    icon: <Group />,
+    layout: 'list',
+    parent: 'Home',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Sessions',
+    icon: <ListIcon />,
+    layout: 'list',
+    parent: 'Performance',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Logs',
+    icon: <UnorderedList />,
+    layout: 'list',
+    parent: 'Performance',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Remote Access',
+    icon: <Location />,
+    layout: 'panes',
+    parent: 'Home',
+  },
+  {
+    name: 'Security',
+    icon: <ShieldSecurity />,
+    layout: 'panes',
+    parent: 'Home',
+  },
+  {
+    name: 'Diagnostics',
+    icon: <Overview />,
+    layout: 'tiles',
+    parent: 'Performance',
+  },
+  {
+    name: 'Server Cluster',
+    icon: <Cluster />,
+    layout: 'tiles',
+    parent: 'Devices',
+    children: ['Server A', 'Server B'],
+  },
+  {
+    name: 'Beacon Cluster',
+    icon: <Cluster />,
+    layout: 'tiles',
+    parent: 'Devices',
+    children: ['Beacon A', 'Beacon B', 'Beacon C', 'Beacon D'],
+  },
+  {
+    name: 'Server A',
+    icon: <Server />,
+    layout: 'panes',
+    parent: 'Server Cluster',
+  },
+  {
+    name: 'Server B',
+    icon: <Server />,
+    layout: 'panes',
+    parent: 'Server Cluster',
+  },
+  {
+    name: 'Beacon A',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon B',
```

Comment:
I noticed that when I click on Devices, the Beacon B card gets automatically focused for some reason:
<img width="1236" alt="Screen Shot 2020-05-06 at 9 25 36 AM" src="https://user-images.githubusercontent.com/12522275/81203012-3f5d0980-8f7c-11ea-819a-f7ceb953b601.png">

Reply:
Seems like the page is reading that card as the same card I had just clicked on or something?